src/utils/triton.h

```diff
@@ -90,6 +99,25 @@ inline TRITONSERVER_MemoryType to_triton(device_type_t dev) {
   }
 }
 
+class TritonError : public UniqueHandle<TRITONSERVER_Error *, TritonError>, public std::exception {
```

Comment:
How about:
```suggestion
class TritonError : public UniqueHandle<TRITONSERVER_Error *, TritonError>, public std::runtime_error {
```
? Just asking, I'm not sure what kind of error it can be.